administration/config/babel.config.ts

```diff
@@ -0,0 +1,114 @@
+const path = require('path')
+
+const validateBoolOption = (option: string, opts: Record<string, unknown>, defaultValue: boolean): boolean => {
+  if (!(option in opts)) {
+    return defaultValue
+  }
+  const value = (opts as Record<string, unknown>)[option]
+  if (typeof value !== 'boolean') {
+    throw Error(`Expected ${value} to be a boolean. Received ${value}`)
+  }
+  return value
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const create = (api: unknown, opts: Record<string, unknown> | null | undefined) => {
+  const env = process.env.NODE_ENV
+  if (env !== 'development' && env !== 'production' && env !== 'test') {
```

Comment:
❓ why we need here `test` ?
We don't distinguish anywhere for this?

Reply:
It seems CRA used to use babel for jest too. As we don't do that anymore, I removed the functionality from these babel configs, and also simplified the babel configs further.